packages/storybook/stories/ImageGallery/snippets/ImageGallery.snippet.tsx

```diff
@@ -0,0 +1,36 @@
+import { ImageGallery, ImageGalleryImageProps, FluentThemeProvider } from '@azure/communication-react';
+import React, { useState } from 'react';
+
+export const ImageGalleryExample: () => JSX.Element = () => {
+  const [galleryImages, setGalleryImages] = useState<Array<ImageGalleryImageProps> | undefined>(undefined);
+  const imgClickedHandler = (event: React.MouseEvent<HTMLImageElement>): void => {
+    event.stopPropagation();
+
+    const img = event.currentTarget;
+    const source = img.src;
+    const title = 'Image';
+    const galleryImage: ImageGalleryImageProps = {
+      title: title,
+      saveAsName: source,
+      imageUrl: source
+    };
+    setGalleryImages([galleryImage]);
+  };
+
+  return (
+    <FluentThemeProvider>
+      <div style={{ width: '31.25rem' }}>
+        <img alt="image" src="images/inlineImageExample1.png" onClick={imgClickedHandler} />
+      </div>
```

Comment:
I don't think we need to display a placeholder image to be clicked on.
I think we can directly display the ImageGallery component without needing to click on anything.

Reply:
> I'm not sure how this will work. The image gallery is a modal so it will take over the entire screen. We will need an onClick event to launch the gallery in some way or else it would be just shown all the time.

That's a good point. Do you think having a button of "Open Image Gallery" would be more clear in this case than having an clickable image?
Contoso might be confused that the placeholder image is the gallery itself since the convention of storybook page is to display the component directly.

Reply:
Discussed offline